packages/fe-container/src/components/footer-bar/FooterBar.less

```diff
@@ -0,0 +1,55 @@
+@import "../../styles/_variables";
+
+.footerBar_container {
+  width: 100%;
+  height: @footer-bar-height;
+  display: flex;
+  background: white;
+
+  & .cookies_svg {
+    float: left;
+    margin-top: 25px;
+    margin-left: 24px;
+    margin-right: 24px;
+    & .svg_primaryColor {
+      fill: var(--microlc-primary-color);
+    }
+
+    & .svg_primaryColorTint89 {
+      fill: var(--microlc-primary-color-tint-89);
+    }
+  }
+
+  & .footerBar_rightSide {
+    display: flex;
+    flex-direction: column;
+    float: left;
+    margin-top: 24px;
+
+    & .banner_text {
+      line-height: 26px;
+      display: flex;
+      flex-direction: column;
+    }
+    a {
+      color: var(--microlc-primary-color);
+    }
+    & .footerBar_buttons {
+      margin-bottom: 24px;
+      margin-top: 24px;
+      .accept_button {
+        background: var(--microlc-primary-color);
+        border-color: var(--microlc-primary-color-tint-89);
+        margin-right: 24px;
```

Comment:
you can put this margin-right and border-radius in a ```button```, to avoid repetition in these 2 classes